libs/blocks/article-feed/article-helpers.js

```diff
@@ -321,3 +316,42 @@ export function stamp(message) {
     console.warn(`${new Date() - performance.timeOrigin}:${message}`);
   }
 }
+
+async function getDocument(path) {
+  const resp = await fetch(path);
+  if (!resp || !resp.ok) {
+    // eslint-disable-next-line no-console
+    console.log(`Could not retrieve metadata for ${path}`);
+    return null;
+  }
+
+  const text = await resp.text();
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(text, 'text/html');
+  return doc;
+}
+
+export async function getBlogArticle(path) {
+  const doc = await getDocument(path);
+
+  if (!doc) return null;
+
+  let title = getMetadata('og:title', doc).trim();
+  const trimEndings = ['|Adobe', '| Adobe', '| Adobe Blog', '|Adobe Blog'];
```

Comment:
`const` should be placed before `let`. Do you have the eslint extension for vscode?
```suggestion
  const trimEndings = ['|Adobe', '| Adobe', '| Adobe Blog', '|Adobe Blog'];
  let title = getMetadata('og:title', doc).trim();
```